tags/tags.json

```diff
@@ -77,5 +77,10 @@
     "defi": {
         "name": "Defi",
         "description": null
+    },
+    "perpetuals": {
+        "name": "Perpetuals",
+        "description": "Tokens that trustlessly recreate decentralized perpetual contracts on other assets",
+        "emoji":"🔄"
```

Comment:
We considered this tag, but do not think that it adds value at this time. Please delete this addition